cmd/kubeadm/app/cmd/init.go

```diff
@@ -418,8 +418,13 @@ func (i *Init) Run(out io.Writer) error {
 	}
 
 	// NOTE: flag "--dynamic-config-dir" should be specified in /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
-	// This feature is disabled by default, as it is alpha still
+	// This feature is disabled by default
 	if features.Enabled(i.cfg.FeatureGates, features.DynamicKubeletConfig) {
+		kubeletVersion, err := preflight.GetKubeletVersion(utilsexec.New())
```

Comment:
? - this seems misplaced to me.

Reply:
kubeletVersion is needed for the dynamic config func. This execs out to the kubelet to determine what version to target. This func call just moved to where it's actually used